ooni/nettests/blocking/web_connectivity.py

```diff
@@ -0,0 +1,512 @@
+# -*- encoding: utf-8 -*-
+
+import csv
+import json
+from urlparse import urlparse
+
+from ipaddr import IPv4Address, AddressValueError
+
+from twisted.web.client import GzipDecoder
+from twisted.internet import reactor
+from twisted.internet.protocol import Factory, Protocol
+from twisted.internet.endpoints import TCP4ClientEndpoint
+from twisted.names import client, dns
+
+from twisted.internet import defer
+from twisted.python import usage
+
+from ooni import geoip
+from ooni.utils import log
+
+from ooni.backend_client import WebConnectivityClient
+
+from ooni.utils.net import StringProducer, BodyReceiver
+from ooni.templates import httpt, dnst
+from ooni.errors import failureToString
+
+REQUEST_HEADERS = {
+    'User-Agent': ['Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, '
+                   'like Gecko) Chrome/47.0.2526.106 Safari/537.36'],
```

Comment:
The issue with this is that we need to ensure that it's also kept in sync with the user-agent that is used by the backend.

I think as long as it's not something that is insanely old it should be fine and we can just update them every year or so.